post/views.py

```diff
@@ -160,6 +172,78 @@ def delete(self, request, post_id):
         return JsonResponse({'message': 'SUCCESS'}, status=200)
 
 
+    @transaction.atomic
+    @login_decorator
+    def put(self, request, post_id):
+        user            = request.user
+        data            = json.loads(request.body)
+        linked_products = data.get('linked_products') #dictionary -option
+
+        if not 'content' in data or not 'images' in data or not 'tags' in data:
```

Comment:
문제는 없지만 아래처럼 수정해주시는것이 가독성에는 좋습니다.
```suggestion
        if 'content' not in data or 'images' not in data or 'tags' not in data:
```
그리고 해당 `key` 값들이 없으면 `keyerror` 를 띄워주고 싶은거라면 이런 `if` 문을 따로 만들어줄 필요없이 `data` 에서 직접 꺼내오게끔 코드를 작성해두면 알아서 `KeyError` exception 을 raise 할거라서 굳이 필요없는 부분이라고 생각합니다.